### DISCUSSION TITLE
Storing JSON Files on Spike Prime

### DISCUSSION THREAD

**Discussion by DylanBPY** [pybricks/support#1805] [Q&A] [answered]
Hello all!
I'm working on a project that involves reading a file from the Spikes files. I have this successfully working on the EV3 since it's easy to download and upload files to it using the ev3dev VS Code extension.

I simply need to be able to upload JSON files to the Spike so it can be read by a file and used.
Any ideas?

**Comment by dlech** [comment, marked as answer]
We don't have file system support on SPIKE Prime. But since Python's `dict` and `list` are very similar to JSON you should be able to copy and paste the JSON into Python files (with some small modifications to assign it to a variable) and make it work that way.

**Reply by DylanBPY** [reply]
Thanks for the quick reply!
I think I should be able to make this method work. 
What is the purpose of the ujson module on Spike then?

**Reply by dlech** [reply]
I suppose `ujson` could be useful if you wanted to receive structured data via `stdin` or send it via `stdout`.

**Reply by DylanBPY** [reply]
Oh, that makes sense, thanks!
